openpype/hosts/maya/plugins/load/load_arnold_standin.py

```diff
@@ -156,12 +166,12 @@ def _setup_proxy(self, shape, path, namespace):
         )
         cmds.setAttr(
             string_replace_operator + ".match",
-            proxy_basename,
+            proxy_basename.split(".")[0],
             type="string"
         )
         cmds.setAttr(
             string_replace_operator + ".replace",
-            os.path.basename(path),
+            os.path.basename(path).split(".")[0],
```

Comment:
Do we need this same fix in the `update` method?

Reply:
Nice spot! Fixed.